cdhweb/pages/admin.py

```diff
@@ -0,0 +1,13 @@
+from django.contrib import admin
+from wagtail.core.models import Page, Site
+from wagtail.documents.models import Document
+from wagtail.images.models import Image
+
+# unregister wagtail content from django admin to avoid
+# editing something in the wrong place and potentially causing
+# problems
+
+admin.site.unregister(Page)
+admin.site.unregister(Site)
+admin.site.unregister(Image)
+admin.site.unregister(Document)
```

Comment:
Is this necessary? Does wagtail register them by default?

Reply:
I checked and it does (although I don't see `Page` here, so I guess we can leave that one out).

<img width="509" alt="Screen Shot 2020-12-21 at 2 50 22 PM" src="https://user-images.githubusercontent.com/4924494/102816339-de176880-439b-11eb-9054-5ce877888e42.png">

Reply:
Looks like it's just documents and images, actually. But I have no idea where Comments comes from...is that a Mezzanine thing?

Reply:
Not sure! "generic" isn't very helpful, is it. I don't see any mention of it in django docs; I think mezzanine has some commenting support but I don't think we would have turned it on.

Reply:
oh - it looks like it's a mezzanine generic http://mezzanine.jupo.org/docs/_modules/mezzanine/generic/models.html

Reply:
ok - I'll leave it alone since it'll go away when mezzanine does, then.